cmd/minikube/cmd/status.go

```diff
@@ -366,7 +369,12 @@ func nodeStatus(api libmachine.API, cc config.ClusterConfig, n config.Node) (*St
 
 	stk := kverify.ServiceStatus(cr, "kubelet")
 	st.Kubelet = stk.String()
-
+	if cc.ScheduledStop != nil {
+		initiationTime := time.Unix(cc.ScheduledStop.InitiationTime, 0)
+		st.TimeToStop = time.Until(initiationTime.Add(cc.ScheduledStop.Duration)).String()
+	} else {
```

Comment:
We can actually remove this else statement since the default for `TimeToStop` is set as `Nonexistent`